cli/lsp/tsc.rs

```diff
@@ -1071,17 +1105,20 @@ fn get_text(state: &mut State, args: Value) -> Result<Value, AnyError> {
   let mark = state.state_snapshot.performance.mark("op_get_text");
   let v: GetTextArgs = serde_json::from_value(args)?;
   let specifier = ModuleSpecifier::resolve_url(&v.specifier)?;
-  let content = if state.state_snapshot.documents.contains(&specifier) {
-    cache_snapshot(state, v.specifier.clone(), v.version.clone())?;
-    state
-      .snapshots
-      .get(&(v.specifier.into(), v.version.into()))
-      .unwrap()
-      .clone()
-  } else {
-    let sources = &mut state.state_snapshot.sources;
-    sources.get_text(&specifier).unwrap()
-  };
+  let content =
+    if let Some(Some(content)) = state.state_snapshot.assets.get(&specifier) {
+      content.text.clone()
+    } else if state.state_snapshot.documents.contains(&specifier) {
```

Comment:
Suggestion: rename `Documents::contains()` to `Documents::contains_key()` for consistency.

(Also makes it clearer that it's a key search, not a string search, which I initially thought it was.)

Reply:
why, I might was well do `Sources::contains()` to `Sources::contains_key()`... 🥳